src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1764,8 +1779,8 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
                     $this->sId = $uid;
                 }
                 --$iMaxTry;
-            } while ($iMaxTry > 0 && !$bWasInserted);
-        } else {
+            } while ($iMaxTry > 0 && false === $bWasInserted);
+        } else { // update query
```

Comment:
It updates a query or it is an update query?

Reply:
We changed the comment to make it clearer